pages/bookmarks.jsx

```diff
@@ -5,6 +5,7 @@ import ProjectList from '../components/project-list/project-list.jsx';
 import HintMessage from '../components/hint-message/hint-message.jsx';
```

Comment:
refactor/clean up work for this file will be done as a separate PR: https://github.com/mozilla/network-pulse/issues/429